api/src/services/sessionService.js

```diff
@@ -0,0 +1,55 @@
+import { db } from '../db';
+
+/**
+ *  Creates new session for new login.
+ *
+ * @param {*} userParams
+ * @returns {Promise}
+ */
+export const createSession = (userParams) => {
+  const sessionRef = db.ref(`/session/${userParams.user.uid}`);
+
+  return sessionRef.set({
+    username: userParams.user.name,
+    createdAt: new Date(),
+    refreshToken: userParams.tokens.refreshToken,
+  });
+};
+
+/**
+ * Deletes Session after logout.
+ *
+ * @param {*} userParams
+ * @returns
+ */
+export const deleteSession = (userId) => {
+  const sessionRef = db.ref(`/session/${userId}`);
+
+  return sessionRef.remove();
+};
+
+/**
+ * Get session of loggged in user.
+ *
+ * @param {*} refreshToken
+ * @returns
+ */
+export const getSession = async (refreshToken) => {
+  const sessionRef = db.ref(`/session`);
+
+  try {
+    const snapshot = await sessionRef.once('value');
+
+    const sessions = snapshot.val();
+
+    const index = Object.keys(sessions).find((key) => {
+      return sessions[key].refreshToken === refreshToken;
+    });
+
+    if (index) {
```

Comment:
there is no else case?

Reply:
No, it return session if it finds otherwise not